### DISCUSSION TITLE
software/2024/12/10/linux-perf-pmu/

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/blog-source#16] [General]
# software/2024/12/10/linux-perf-pmu/

杰哥的{运维，编程，调板子}小笔记

https://jia.je/software/2024/12/10/linux-perf-pmu/

**Comment by myl7** [comment]
最后一段的标题似乎是有 typo？

**Reply by jiegec** [reply]
感谢提醒，已修复。
